source/developers-guide/shopware-composer/index.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: default
+title: Using Composer to install Shopware
+github_link: developers-guide/shopware-composer/index.md
+indexed: true
+group: Developer Guides
+subgroup: General Resources
+menu_title: Using composer with Shopware
+menu_order: 45
```

Comment:
should be higher imho. Right after Cheat-Sheet and before PhpStorm. something like 2 or 3